src/components/IconGithub.jsx

```diff
@@ -1,16 +1,44 @@
+import React, { useState, useEffect } from 'react';
+
 export default function IconGithub() {
+  const [stars, setStars] = useState(0);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(`https://api.github.com/repos/markmead/hyperui`);
+        const data = await response.json();
+        const starCount = data.stargazers_count;
+        setStars(starCount);
+      } catch (error) {
+        console.error('Error fetching star count:', error);
+      }
+    };
+
+    fetchData();
+  }, []);
+  const formatStarCount = (count) => {
+    if (count >= 1000) {
+      return (count / 1000).toFixed(1) + 'K';
+    }
+    return count;
+  };
```

Comment:
Let's see how it looks without this formatting.

Reply:
removed the formatting... the idea behind doing it was to make the header look clean and keep the increasing star count under control.